### PR TITLE
Create template for prom-alertmanager docker container

### PR DESCRIPTION
**Pull request by clintonsteiner**
Application install for this was missing so went ahead and created it based off the official dockerfile and image.
Tested locally but first time creating a template so let me know if you see any issues.
This is the guide I followed.
https://selfhosters.net/docker/templating/templating/

### CODE REVIEW

File: templates/prom-alertmanager.xml
Comment (nwithan8) [inline templates/prom-alertmanager.xml:45]: We'll use `/mnt/user/app data/alertmanager` as the root directory on Unraid to store all things related to the container, although this path doesn't directly map to anything inside the container. So this `<Config>` element is correct, good job!

Let's add a second path that maps `/mnt/user/app data/alertmanager/storage` to [`/alertmanager`](https://github.com/prometheus/alertmanager/blob/c920b605b647f2548e23a3dc2fea014b9c9dbf43/Dockerfile#L17) inside the container

File: templates/prom-alertmanager.xml
Comment (nwithan8) [inline templates/prom-alertmanager.xml:38]: Let's simplify this a bit. The `Target` is correct (`/etc/alertmanager/alertmanager.yaml`), but we don't need to replicate that directory path exactly on the host. Instead, make the mapping (and `Default`) so that `/mnt/user/app data/alertmanager/alertmanager.yaml` maps to `/etc/alertmanager/alertmanager.yaml` inside the container.

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: templates/prom-alertmanager.xml
Comment (clintonsteiner) [reply templates/prom-alertmanager.xml:50]: Fixed

File: templates/prom-alertmanager.xml
Comment (clintonsteiner) [reply templates/prom-alertmanager.xml]: Fixed

Comment (clintonsteiner) [reply templates/prom-alertmanager.xml]: Wait no

Comment (clintonsteiner) [reply templates/prom-alertmanager.xml]: <Config Name="ConfigFile" Target="/etc/alertmanager/alertmanager.yaml"
    Default="/mnt/user/appdata/alertmanager/alertmanager.yaml" Mode="rw"
    Description="Config directory" Type="Path" Display="advanced" Required="false" Mask="false">
    /mnt/user/appdata/alertmanager/alertmanager.yaml</Config>

File: templates/prom-alertmanager.xml
Comment (nwithan8) [inline templates/prom-alertmanager.xml:38]: The Target on this needs to be `/alertmanager`

**Review by nwithan8 — CHANGES_REQUESTED** [review]

File: templates/prom-alertmanager.xml
Comment (clintonsteiner) [reply templates/prom-alertmanager.xml]: resolved

**Comment by clintonsteiner** [comment]
@nwithan8 thanks for taking the time to review. Appreciate you helping me get this polished up before merging

**Review by nwithan8 — APPROVED** [review]
LGTM, great work!

**Comment by clintonsteiner** [comment]
@nwithan8 Looks like you need write access to merge pr's. Are you able to merge this or do we need to wait for another reviewer to come by?

File: templates/prom-alertmanager.xml
Comment (Roxedus) [inline templates/prom-alertmanager.xml:50]: Whats this for?

File: templates/prom-alertmanager.xml
Comment (Roxedus) [inline templates/prom-alertmanager.xml:36]: There is no reason to set a value here when using Default

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: templates/prom-alertmanager.xml
Comment (clintonsteiner) [reply templates/prom-alertmanager.xml:50]: Unsure, will remove

File: templates/prom-alertmanager.xml
Comment (clintonsteiner) [reply templates/prom-alertmanager.xml:36]: Makes sense to me, going to get rid of the 9093

File: templates/prom-alertmanager.xml
Comment (clintonsteiner) [reply templates/prom-alertmanager.xml]: Fixed

File: templates/prom-alertmanager.xml
Comment (clintonsteiner) [reply templates/prom-alertmanager.xml]: Fixed

**Comment by clintonsteiner** [comment]
@Roxedus appreciate you taking the time to review. Apologies for taking some time getting back to this, was away over the holidays

**Review by Roxedus — APPROVED** [review]

**Pull request merged**
